chat-commands.js

```diff
@@ -70,6 +70,16 @@ exports.commands = {
 		"/auth [room] - Show what roomauth a room has.",
 		"/auth [user] - Show what global and roomauth a user has."],
 
+	userlist: function (target, room, user) {
+		let userlist = room.getUserList().substr(7);
+		userlist = userlist.split(',').splice(1); // split userlist by comma, exclude usercount
```

Comment:
> `splice`

...???????

`splice` is a very unperformant (you shouldn't ever use it unless you need to mutate an array for whatever reason, usually `slice`/`concat` is better) and isn't even doing anything here (`splice` needs at least two arguments)